[2025-04-09T07:11:32.290+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.export_to_csv manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:32.300+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.export_to_csv manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:32.300+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T07:11:32.372+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): export_to_csv> on 2025-04-09 07:11:12.014769+00:00
[2025-04-09T07:11:32.379+0000] {standard_task_runner.py:60} INFO - Started process 193 to run task
[2025-04-09T07:11:32.382+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'export_to_csv', 'manual__2025-04-09T07:11:12.014769+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpvtlvn9m0']
[2025-04-09T07:11:32.385+0000] {standard_task_runner.py:88} INFO - Job 85: Subtask export_to_csv
[2025-04-09T07:11:32.555+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.export_to_csv manual__2025-04-09T07:11:12.014769+00:00 [running]> on host 6d74a53745cf
[2025-04-09T07:11:32.798+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='export_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T07:11:12.014769+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T07:11:12.014769+00:00'
[2025-04-09T07:11:32.799+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T07:11:32.800+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/export_to_csv.py']
[2025-04-09T07:11:32.814+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T07:11:33.513+0000] {subprocess.py:93} INFO - ‚ùå Error al exportar a CSV: [Errno 2] No such file or directory: '../data/processed/final_merged.parquet'
[2025-04-09T07:11:33.608+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T07:11:33.665+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=export_to_csv, execution_date=20250409T071112, start_date=20250409T071132, end_date=20250409T071133
[2025-04-09T07:11:33.736+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T07:11:33.755+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
